src/telemetry.ts

```diff
@@ -58,7 +72,7 @@ export class GATelemetry implements Telemetry {
     return this._isTelemetryEnabled;
   }
 
-  async setup() {
+  setup() {
     if (!this.isTelemetryEnabled) {
```

Comment:
I think this needs parens:

```suggestion
    if (!this.isTelemetryEnabled()) {
```

Reply:
Ah, I thought already added that but apparently I did not! Good catch, updated!